tests/config/conf_wrong_data_and_sampling.yml

```diff
@@ -23,13 +23,13 @@ algorithm:
     transfer_strategy: Recursive
     criterion: Tempered
     max_objects_per_transfer: 8
-    deterministic_transfer: True
+    deterministic_transfer: true
```

Comment:
Is the capitalized `True` non-standard and to be replaced by `true`?

Reply:
This is the standard way used by pyyaml to dump bool values